File: templates/python/protection_fault_calendar_grid.py
"""protection_fault_calendar_grid: 保护与故障分析日历网格（calendar-grid 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='protection and fault analysis: calendar grid'):
    return make_template_figure('calendar_grid', seed=4019, title=title, domain='protection and fault analysis', topic='calendar grid')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/protection_fault_calendar_grid.m
function fig = protection_fault_calendar_grid()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('calendar_grid', 4019, 'protection and fault analysis: calendar grid', 'protection and fault analysis', 'calendar grid');
end
